pages/data-migration/migrate-from-neo4j.mdx

```diff
@@ -151,9 +157,9 @@ This can be done by copying the file into your running instance.
 
 1. Run Memgraph 
 
-    The easiest way to run Memgraph is by running installation scripts that use a
-    pre-prepared `docker-compose.yml` file by executing one of the following
-    commands, depending on your operating system: 
+    The easiest way to run Memgraph MAGE and Memgraph Lab is by running
+    installation scripts that use a pre-prepared `docker-compose.yml` file by
+    executing one of the following commands, depending on your operating system: 
```

Comment:
Not sure if this is a bit confusing since in the prerequisites part we state that the prerequisite is to have the latest Docker images and then here we're mentioning running installation scripts. 
It would maybe make more sense to add that a prereq is having a running Memgraph instance and then link to the installation in the docs and keep this step the same, mentioning the easiest way to run Memgraph